scripts/ci/libraries/_spinner.sh

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Rather than showing our animated spinner during the build I chose the option to simply show the buildkit output directly in the terminal. It's nice, colorful, animated and optimized to show only relevant information and take as little space as possible during the build. So we can remove the animated spinner now. 

Also during the pre-commit  I opted to just show the warning that image is not updated rather that try to build it. User can build it when entering breeze or by (suggested) manual action. This simplifies interaction in pre-commit (there the difficulty is that the terminal there is difficult to get hold of).